pages/Grind-Stone.md

```diff
@@ -0,0 +1,13 @@
+The Grind Stone is in the [Basic Machines](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Basic-Machines) category. It is a multiblock structure.<br>
+
+## Obtaining
+The Grind Stone is a multiblock structure, it consists of a dispenser and a fence.<br>
+
+A complete setup should look like this:
+
+![Grind Stone Setup](https://raw.githubusercontent.com/TheBusyBiscuit/Slimefun4-Wiki/master/images/multiblock-grind-stone.png)
+
+You can also place an [Output Chest](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Output-Chest) next to the dispenser to collect your resources. If there is no [Output Chest](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Output-Chest), then it will be put in the dispenser.
+
+## Usage
+The Grind Stone can be used to grind items/blocks into other items/blocks.
```

Comment:
This is a wiki, please be more precise than that.
State that it is used to craft various items and that they should look into the Slimefun Guide for more.